frontend/pages/[project]/proposal/index.tsx

```diff
@@ -124,96 +117,20 @@ function ProposalForm() {
 
   return (
     <>
-      <Title
-        title="New Problem"
-        subtitle="Please fill out all the required fields of the form to propose a new problem."
-        topSpacing={7 * GU}
-        bottomSpacing={5 * GU}
+      <CreateProblemOrSolutionForm 
```

Comment:
Probably this approach is not the best one either, as this form may get more complex in the future.
But for the time being, i thought it might be better not to move the states and the call functions down untill we have a more clrear vision, and keepping the form component only handle visual tasks.